packages/agent-toolkit/src/core/tools/platform-api-tools/create-item-tool/create-item-tool.ts

```diff
@@ -0,0 +1,149 @@
+import { z } from 'zod';
+import { CreateItemMutation, CreateItemMutationVariables, DuplicateItemMutation, CreateSubitemMutation } from '../../../../monday-graphql/generated/graphql';
+import { createItem, duplicateItem, createSubitem } from '../../../../monday-graphql/queries.graphql';
+import { ToolInputType, ToolOutputType, ToolType } from '../../../tool';
+import { BaseMondayApiTool, createMondayApiAnnotations } from '../base-monday-api-tool';
+import { ChangeItemColumnValuesTool } from '../change-item-column-values-tool';
+
+export const createItemToolSchema = {
+  name: z.string().describe("The name of the new item to be created, must be relevant to the user's request"),
+  groupId: z
+    .string()
+    .optional()
+    .describe('The id of the group id to which the new item will be added, if its not clearly specified, leave empty'),
+  columnValues: z
+    .string()
+    .describe(
+      `A string containing the new column values for the item following this structure: {\\"column_id\\": \\"value\\",... you can change multiple columns at once, note that for status column you must use nested value with 'label' as a key and for date column use 'date' as key} - example: "{\\"text_column_id\\":\\"New text\\", \\"status_column_id\\":{\\"label\\":\\"Done\\"}, \\"date_column_id\\":{\\"date\\":\\"2023-05-25\\"},\\"dropdown_id\\":\\"value\\", \\"phone_id\\":\\"123-456-7890\\", \\"email_id\\":\\"test@example.com\\"}"`,
+    ),
+    parentItemId: z.number()
+    .optional()
+    .describe('The id of the parent item under which the new subitem will be created'),
+    duplicateFromItemId: z.number()
+    .optional()
+    .describe('The id of existing item to duplicate and update with new values (only provide when duplicating)'),
+};
+
+export const createItemInBoardToolSchema = {
+  boardId: z.number().describe('The id of the board to which the new item will be added'),
+  ...createItemToolSchema,
+};
+
+export type CreateItemToolInput = typeof createItemToolSchema | typeof createItemInBoardToolSchema;
+
+export class CreateItemTool extends BaseMondayApiTool<CreateItemToolInput> {
+  name = 'create_item';
+  type = ToolType.WRITE;
+  annotations = createMondayApiAnnotations({
+    title: 'Create Item',
+    readOnlyHint: false,
+    destructiveHint: false,
+    idempotentHint: false,
+  });
+
+  getDescription(): string {
+    return 'Create a new item with provided values, create a subitem under a parent item, or duplicate an existing item and update it with new values';
```

Comment:
So agent won't confuse, il add a small explanation here of what params are relevant to which flow